styleguide/core/Form.hbs

```diff
@@ -0,0 +1,23 @@
+{{#defineBlock "Form"}}
+
+    {{#defineElement "actionButton"}}
+        <div class="{{elementName}}">{{this}}</div>
+    {{/defineElement}}
+
+    {{#defineElement "items"}}
+        {{#each this}}
+            <div class="{{elementName}}-item">{{this}}</div>
+        {{/each}}
+    {{/defineElement}}
+
+    {{#defineBlockContainer}}
+
+        <form class="{{blockName}}" {{#with actionUrl}} action="{{this}}" {{/with}}>
```

Comment:
I think there are quite a few missing attributes that are common enough to add here: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/form

At a minimum I think we should have `method` and `enctype`